### ISSUE TITLE
As an LTFAT or AMToolbox developer I would like to retain the copyright of my work

### ISSUE THREAD

**Issue by psoenderg** [ltfat/ltfat#31]
Currently, mat2doc works such that the same copyright headline is added to all files, automatically giving the copyright to Peter Søndergaard.

A better system would be to somehow be able to mark each file with the author that should keep the copyright, this would make the projects a more welcoming. 

This is a bigger problem for AMToolbox than for LTFAT, because AMToolbox has a widely scattered group of developers.

**Comment by susnak** [comment]
Should we also move mat2doc here? And as a repository in ltfat organization?

**Comment by jvnv** [comment]
I think that it should at least be stated who the author of a file is. This
should be done in every file which is in LTFAT. This not only gives some
"credit" to the author, but it helps also to find the person that wrote the
file if there is some question about it or if it should be modified. I
think this is clear for most of the developers, but for someone external
this allows the possibility to contact the author of a file directly (in
case this is wished).

**Comment by psoenderg** [comment]
@susnak: Yes, let's move mat2doc as well and make it a reposity in the ltfat organization, wouldn't that be the least amount of work?

**Comment by psoenderg** [comment]
@jvnv: It is already easy to add the author. I think I made it standard that you could write "% AUTHOR: Your Name" below the header, and then there is a part of mat2doc that can check for missing author statements (and other keywords in this manner).

I did it at one point to find files with missing meta-information. There are also some tags named "TEST" and "REFERENCE" to make sure that each file was tested and had a reference implementation (if applicable).

The copyright is more difficult, because mat2doc makes it easy to update the year, so we don't manually have to handedit all files just to keep the year up to date.

**Comment by jvnv** [comment]
Yes, I know these keyword already exist in some/most files. I have included
them with all the applicable keywords with the files I wrote. However,
there are still a lot of files without an author. I referred to including
the author keyword at least in each file, so I would like to propose to add
it also to the existing files that doesn't have an author keyword yet.

Would it be worthwhile to make this a standard and add it to all the files?
